### ISSUE TITLE
Exclude requests by content type to decrease loading time

### ISSUE THREAD

**Issue by malteos** [wkeeling/selenium-wire#50]
Thanks for your work on selenium-wire! However, I'm facing some performance issues. Opening a single webpage sometimes takes +60s and thus results in a timeout. A reason seems to be that selenium-wire processes all non-resources (images, fonts). 

My log file contains a lot of "Capturing response" messages. A possible for a speedup would be to exclude all those requests, i.e., to have a filter for unwanted content types. Is this currently possible? Or what would be a starting point to contribute such a feature?

**Comment by wkeeling** [comment]
@malteos thanks for raising this.

Have you tried switching on the [`disable_encoding`](https://github.com/wkeeling/selenium-wire#misc-options) option? It might be worth trying this initially, as this can significantly improve page load times when pages have been compressed.
```
options = {
    'disable_encoding': True  # Tell the server not to compress the response
}
driver = webdriver.Firefox(seleniumwire_options=options)
```
I haven't yet got to the bottom of why compressed pages sometimes significantly slow down selenium wire (it doesn't happen in every case), but it may be worth trying that option first.

However, your suggestion about filtering request/response capture based on content type is potentially also good idea and may improve performance. If you felt like creating a PR, please feel free. 

Before you start, it may be worth commenting out this line

https://github.com/wkeeling/selenium-wire/blob/23d69eace63a67973807f315cf8d5f7c456c75a0/seleniumwire/proxy/handler.py#L154

and

https://github.com/wkeeling/selenium-wire/blob/23d69eace63a67973807f315cf8d5f7c456c75a0/seleniumwire/proxy/handler.py#L170

and running your test to see what the performance looks like when _nothing_ is captured. If you see no improvement, then it could be that the performance bottleneck is elsewhere.

**Comment by wkeeling** [comment]
@malteos there is a new version available (v1.0.5) which fixes a problem to do with the response output stream not being properly closed. It was observed that this was causing a significant slowdown. If you get a chance to upgrade to 1.0.5 you may discover things run a bit quicker.

**Comment by malteos** [comment]
After upgrading to v1.0.5 and changing from Chrome to FirefoxDriver I managed to resolve my issue. Thanks!
